odxtools/cli/_print_utils.py

```diff
@@ -149,20 +149,20 @@ def extract_parameter_tabulation_data(parameters: List[Parameter]) -> Table:
     table.add_column("Bit Length", justify="right", style="yellow")
     table.add_column("Semantic", justify="left", style="white")
     table.add_column("Parameter Type", justify="left", style="white")
-    table.add_column("Data Type", justify="left", style="yellow")
-    table.add_column("Value", justify="left", style="white")
+    table.add_column("Data Type", justify="left", style="white")
+    table.add_column("Value", justify="left", style="yellow")
     table.add_column("Value Type", justify="left", style="white")
     table.add_column("Linked DOP", justify="left", style="white")
 
-    name: List[Optional[object]] = []
-    byte: List[Optional[object]] = []
-    bit_length: List[Optional[object]] = []
-    semantic: List[Optional[object]] = []
-    param_type: List[Optional[object]] = []
-    value: List[Optional[object]] = []
-    value_type: List[Optional[object]] = []
-    data_type: List[Optional[object]] = []
-    dop: List[Optional[object]] = []
+    name: List[str] = []
+    byte: List[Any] = []
+    bit_length: List[Any] = []
```

Comment:
that should probably be
```suggestion
    byte: List[Optional[int]] = []
    bit_length: List[Optional[int]] = []
```
?